reference/2_constructing_data/2_3_merge_concat/README.md

```diff
@@ -0,0 +1,104 @@
+# Merging & concatenation
+
+## Example - Merging dictionaries
+
+> **_Try out:_**
+>
+> Place node **tutorials/data_constr/merge_dict** or **[download .cranqj](cranqj/data_constr_merge_dict.cranqj)** 
+
+Dictionaries in Cranq can be merged with the ```data/dictionary/Merger``` node.
+- The node simply takes 2 dictionaries
+- If the keys conlict, then the values in "b" take precedence
+
+![](images/2021-07-19-14-23-42.png)
+
+### Sample output:
+
+```json
+{
+  "a": "b",
+  "c": 3,
+  "d": "e"
+}
+```
+
+## Example - Concatenating arrays
+
+> **_Try out:_**
+>
+> Place node **tutorials/data_man/concat_array** or **[download .cranqj](cranqj/data_constr_concat_array.cranqj)** 
+
+Concatenating arrays in Cranq can be performed with the ```data/array/Concatenator``` node.
+
+
+![](images/2021-07-19-14-27-25.png)
+
+- It simply takes 2 arrays & applies the items of "b" onto "a", in order
+- The example combines the node (```tutorials/data_constr/syncer_array_emps```) created in **[Example - Initializing an array with values](../2_4_syncer_splitter/README.md)** & the following test data:
+  ```json
+  [
+    {
+      "EmpID": 001,
+      "Name": "Joan",
+      "Dept": "Accounting",
+      "HireDate": "2016-04-28",
+      "Salary": 2200
+    },
+    {
+      "EmpID": 002,
+      "Name": "Frank",
+      "Dept": "Facilities",
+      "HireDate": "2011-11-13",
+      "Salary": 2000
+    },
+    {
+      "EmpID": 003,
+      "Name": "Louis",
+      "Dept": "Legal",
+      "HireDate": "2015-09-01",
+      "Salary": 3000
+    }
+  ]
+  ```
+
+### Sample output:
+
+```json
+[
+  {
+    "EmpID": 101,
+    "Name": "Sue",
+    "Dept": "Facilities",
+    "HireDate": "2019-02-13",
+    "Salary": 1500
+  },
+  {
+    "EmpID": 100,
+    "Name": "Ted",
+    "Dept": "Accounting",
+    "HireDate": "2020-11-08",
+    "Salary": 1500
+  },
+  {
+    "EmpID": 1,

```

Comment:
EmpID should be 001

Reply:
Maybe better to change the original EmpID to 1 since the dataset is used that way in further examples.

Reply:
Will fix, nice catch